src/server.c

```diff
@@ -5911,25 +5911,25 @@ void sendChildInfo(childInfoType info_type, size_t keys, char *pname) {
 
 /* Use 'MADV_DONTNEED' to release memory to operation system quickly. */
 void redisUnusedMemory(void *ptr) {
+#if HAVE_MALLOC_SIZE
```

Comment:
we do have HAVE_MALLOC_SIZE on glibc malloc too.
```suggestion
#if USE_JEMALLOC
```

btw, i think it's odd this method is placed in server.c, maybe we should move it to zmalloc.c (and find a way to init the page_size global just once)